BUILD.gn

```diff
@@ -26,6 +26,10 @@ import("${chip_root}/gn/chip/tests.gni")
 import("${chip_root}/gn/chip/tools.gni")
 
 if (current_toolchain != "${dir_pw_toolchain}/dummy:dummy") {
+  declare_args() {
+    chip_enable_python_modules = true
```

Comment:
chip_enable_python_modules = current_os == "linux"

Reply:
Previous code was adding it twice: with current os and without, so I preserved the 'added always'. Is current_os more correct?

Reply:
No, I didn't see the other one. OK, I guess this is fine because embedded are building tools and I think it builds on Mac now.